modules/mobile/app/services/mobile/v0/appointments/appointment_requests_service.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module Mobile
+  module V0
+    module Appointments
+      # connects to appointment request service
+      class AppointmentRequestsService < VAOS::SessionService
+        # fetches the past 90 days of appointment requests
+        def get_appointment_requests
+          response = nil
+          error = nil
+          end_date = Time.zone.today
+          start_date = end_date - 90.days
+          date_params = { startDate: date_format(start_date), endDate: date_format(end_date) }
+
+          with_monitoring do
```

Comment:
This is directly cribbed from the vaos code.

Reply:
This adds some metrics automatically behind the scenes. Not sure that we need it as long as we [register our service with the breakers gem](https://github.com/department-of-veterans-affairs/vets-api/blob/1ce184eae8c63b9ff8142ad1ba8c6350b1e2e5cd/config/initializers/breakers.rb#L80). That enables metrics automatically for the service and it shows up in Datadog dashboards (assuming we keep the service rather than re-use VAOS's).

Reply:
I'll look into that. I'd also prefer to re-use code.